CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+# Version 2.2.0
+- Introduces SpringBoot Starter 1.0.2-BETA
+- Starter now respects autoconfiguration for Micrometer metrics.
+- Fixes the thread shutdown issue in SpringBoot Starter by registering `ApplicationInsightsServletContextListener`.
```

Comment:
can you please add a link to the GH issue with the shutdown?

Reply:
Done